app/views/test_executions/results/_supplemental_data_error_table.html.erb

```diff
@@ -15,7 +15,7 @@
       <tr>
         <th colspan = '5' scope = 'col' class = 'col-sm-2'>
           <% if pop_error_value %>
-            <% if population_errors %>
+            <% unless population_errors.empty? %>
```

Comment:
can `population_errors` ever be `nil`? If it is, the second case will blow up, because `nil` doesn't respond to `.empty?`. Maybe we need something like `if population_errors && !population_errors.empty?`